### PR TITLE
Change test DataLoader to use the test batch size

### PR DESCRIPTION
**Pull request by rdinse**
The batch size for the training set was erroneously used for instantiating the DataLoader class for the test set.

### CODE REVIEW

**Comment by soumith** [comment]
thanks!
